File: 841-shortest-distance-to-a-character/shortest-distance-to-a-character.js
/**
 * @param {string} s
 * @param {character} c
 * @return {number[]}
 */
var shortestToChar = function(s, c) {
    const arr = []
    for (let i=0;i<s.length;i++){
        if(s[i]===c){
            arr.push(i)
        }
    }
    const result = []
     for (let i=0;i<s.length;i++){
        let miin = []
        for(let j of arr){
            miin.push(Math.abs(i-j))
        }
        result.push(Math.min(...miin))
    }

    return result
};

File: 841-shortest-distance-to-a-character/shortest-distance-to-a-character.py
class Solution:
    def shortestToChar(self, s: str, c: str) -> List[int]:
        arr = [i for i in range(len(s)) if s[i] == c]
        res = []
        for i in range(len(s)):
            miin = []
            for j in arr:
                miin.append(abs(i-j))
            res.append(min(miin))
        return res